static/data-ingestion.yaml

```
@@ -169,93 +104,178 @@ paths:
+      operationId: ProductMetadataPatch
       parameters:
-        - $ref: '#/components/parameters/DATA_SPACE_ID'
-        - $ref: '#/components/parameters/ContentType'
-        - $ref: '#/components/parameters/xApiKey'
-        - $ref: '#/components/parameters/xGwSignature'
-        - $ref: '#/components/parameters/ContentEncoding'
+        - name: DATA_SPACE_ID
+          in: path
+          description: Data Space ID
+          required: true
+          style: simple
+          schema:
+            type: string
+        - name: Content-Type
+          in: header
+          required: true
+          schema:
+            type: string
+            enum: [ application/json ]
+        - name: x-api-key
+          in: header
+          required: true
+          schema:
+            type: string
+          description: Production public API key
+        - name: x-gw-signature
+          in: header
+          required: true
+          schema:
+            type: string
+          description: JWT generated for Production public API key
+        - name: Content-Encoding
+          in: header
+          required: false
+          schema:
+            type: string
+            enum: [ gzip ]
+          description: Use this header if the payload is compressed with gzip
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/FeedMetadata'
+            examples:
+              FeedWithMetadataInformation:
+                summary: Example of product attributes metadata feed
+                description: Enables the feed with product attribute information that needs to be updated
+                value:
+                  [
+                    {
+                      "code": "name",
+                      "scope": {
+                        "locale": "En"
+                      },
+                      "label": "Updated - Product Name",
+                      "dataType": "SELECT",
+                      "roles": [ "PDP", "PLP" ],
+                      "filterable": true,
+                      "sortable": true,
+                      "searchable": true,
+                      "searchWeight": 77,
+                      "searchTypes": [ "STARTS_WITH", "AUTOCOMPLETE" ]
+                    }
+                  ]
       responses:
-        '202':
-          $ref: '#/components/responses/AcceptedResponse'
+        '200':
+          description: Request accepted. Check the "invalidFeedItems" node for any invalid items that were not consumed
+          content:
+            application/json;charset=UTF-8:
+              schema:
+                $ref: '#/components/schemas/ProcessFeedResponse'
         '400':
-          $ref: '#/components/responses/InvalidItemsResponse'
+          description: All feed items received were not valid. Check response body for errors
```

for CCDM API entire request will be failed if a t least 1 item is invalid / didn't pass validation 

```suggestion
          description: Some of the received items are not valid. Check the "invalidFeedItems" node for errors
```

---

Done